bark/tools/dat-from-wav.py

```diff
@@ -0,0 +1,28 @@
+from scipy.io import wavfile as wavfile
+import bark, os
+
+
+def main(wav):
+	if not isinstance(wav, str):
```

Comment:
It's usually more idiomatic in python to avoid argument validation, and simply assume the object you're handed is of the proper type.